contributors/guide/first-contribution.md

```diff
@@ -19,69 +19,113 @@ description: |
+is useful to read and understand.
+
+There is an entire SIG ([sig-contributor-experience]) devoted to improving your 
+experience as a contributor.
 
-#### Find a SIG that is related to your contribution
+If you have an idea for improving the contributor experience, please consider
+attending one of the Contributor Experience SIG's [bi-weekly meetings].
 
-Finding the appropriate SIG for your contribution and adding a SIG label will help you ask questions in the correct place and give your contribution higher visibility and a faster community response.
+### Find a SIG that is related to your contribution
 
-For Pull Requests, the automatically assigned reviewer will add a SIG label if you haven't done so. See [Open A Pull Request](#open-a-pull-request) below.
+Finding the appropriate SIG for your contribution and adding a SIG label will 
+help you ask questions in the correct place and give your contribution higher 
+visibility and a faster community response.
 
-For Issues, we are still working on a more automated workflow.
-Since SIGs do not directly map onto Kubernetes subrepositories, it may be difficult to find which SIG your contribution belongs in.
-Here is the [list of SIGs](/sig-list.md) so that you can determine which is most likely related to your contribution.
+For Pull Requests, the automatically assigned reviewer will add a SIG label 
+if you haven't already done so. 
 
-*Example:* if you are filing a CNI issue (that's [Container Networking Interface](https://github.com/containernetworking/cni)), you should choose the [Network SIG](http://git.k8s.io/community/sig-network). Add the SIG label in a comment like so:
+For Issues, please note that the community is working on a more automated workflow.
+
+Since SIGs do not directly map onto Kubernetes subrepositories, it may be 
+difficult to find which SIG your contribution belongs in.
+
+Review the [list of SIGs][sl]] to determine which SIG is most likely related to your 
+contribution.
+
+*Example:* if you are filing a CNI issue (that's [Container Networking Interface]) 
+you'd choose the [Network SIG]. Add the SIG label in a new comment on GitHub 
+by typing the following:
 ```
 /sig network
 ```
 
-Follow the link in the SIG name column to reach each SIGs README. 
-Most SIGs will have a set of GitHub Teams with tags that can be mentioned in a comment on issues and pull requests for higher visibility. 
-If you are not sure about the correct SIG for an issue, you can try SIG-contributor-experience [here](/sig-contributor-experience#github-teams), or [ask in Slack](http://slack.k8s.io/).
+Follow the link in the SIG name column to reach each SIG'ss README. 
+
+Most SIGs will have a set of GitHub Teams with tags that can be mentioned in a 
+comment on issues and pull requests for higher visibility. 
+If you are not sure about the correct SIG for an issue, you can try 
+SIG-contributor-experience [here], or [ask in Slack].
```

Comment:
```suggestion
Most SIGs will have a set of GitHub Teams with tags that can be mentioned in a 
comment on issues and pull requests for higher visibility.  If you are not sure
about the correct SIG for an issue, you can try [SIG contributor-experience], or [ask in Slack].
```
Consolidated and removed the `[here]`, the actual text itself can serve as a link :) I updated it below in another suggestion.